src/Compilers/Core/Desktop/CommandLine/RuleSet/RuleSetProcessor.cs

```diff
@@ -56,26 +58,26 @@ public static RuleSet LoadFromFile(string filePath)
             filePath = FileUtilities.NormalizeAbsolutePath(filePath);
             XmlReaderSettings settings = GetDefaultXmlReaderSettings();
 
-            XmlDocument ruleSetDocument = new XmlDocument();
-            XmlNode ruleSetNode = null;
+            XDocument ruleSetDocument = null;
+            XElement ruleSetNode = null;
 
             using (FileStream stream = FileUtilities.OpenRead(filePath))
             using (XmlReader xmlReader = XmlReader.Create(stream, settings))
             {
                 try
                 {
-                    ruleSetDocument.Load(xmlReader);
+                    ruleSetDocument = XDocument.Load(xmlReader);
                 }
-                catch (XmlException e)
+                catch (Exception e)
                 {
                     throw new InvalidRuleSetException(e.Message);
                 }
 
                 // Find the top level rule set node
-                XmlNodeList nodeList = ruleSetDocument.GetElementsByTagName(RuleSetNodeName);
+                List<XElement> nodeList = ruleSetDocument.Elements(RuleSetNodeName).ToList();
                 if (nodeList.Count != 1)
                 {
-                    Debug.Fail("Multiple top-level nodes!");
+                    // Debug.Fail("Multiple top-level nodes!");
```

Comment:
@bkoelman good catch.  Yes this was unintentional.  Was testing out something else and forgot to revert that.